src/core/QCheck2.ml

```diff
@@ -1430,10 +1436,11 @@ module Test = struct
   let default_if_assumptions_fail = `Warning, 0.05
 
   let make_cell ?(if_assumptions_fail=default_if_assumptions_fail)
-      ?(count) ?(long_factor=1) ?max_gen
+      ?(count) ?long_factor ?max_gen
       ?(max_fail=1) ?(retries=1) ?(name=fresh_name()) ?print ?collect ?(stats=[]) gen law
     =
     let count = global_count count in
+    let long_factor = global_long_factor long_factor in
```

Comment:
I would have expected similar changes below in `make_cell_from_QCheck1`.
Otherwise I suspect the QCHECK_LONG_FACTOR environment variable will only work for QCheck2 tests - but not for QCheck tests.

Reply:
`QCheck.Test.make_cell` calls `QCheck2.Test.make_cell_from_QCheck1` to map all QCheck(1) tests to QCheck2 tests, hence my concern.

Reply:
Thanks you're right, that was a mistake I thought it was covered (but tbh I did not look very deeply)

Reply:
Should be fine like this